tests/test_views.py

```diff
@@ -51,7 +50,9 @@ def test_query_matching_authors(self):
         # Create ten citations to match the query and one that doesn't.
         factories.CitationFactory.create_batch(10, author='Verne, Jules')
         factories.CitationFactory(author='Dan Longshot')
```

Comment:
Okay. There may be other views to do a similar test on then--and some should be expected to fail as noted in other comment.